scripts/markets/help.ts

```diff
@@ -193,7 +191,7 @@ export async function createAndSendMarketMapProposal(
 
       if (config.adjustByMarket) {
         const [Base, Quote] = config.adjustByMarket.split('-');
-        normalize_by_pair = { Base, Quote };
+        normalize_by_pair = { Base: Base!, Quote: Quote! };
```

Comment:
I think this is fine for this random script that is going away iirc. Either way it's gonna crash, most this would do is provide a nicer error message.